lib/plugins/aws/deploy/lib/checkForChanges.js

```diff
@@ -107,7 +108,7 @@ module.exports = {
 
     return BbPromise.all(getFunctionResults).then((results) => {
       if (couldNotAccessFunction) {
-        this.serverless.cli.log(
+        legacy.log(
```

Comment:
I think this should be left untouched, or come with a modern warning counterpart.

As this change signals, we don't want to show this information anymore, and I wonder if that's really the case (?)

Reply:
I think we can leave it untouched for now and switch to warning as a part of separate PR (my intention was to configure warning in new-style as a separate PR)